stress_test/emulator.py

```diff
@@ -0,0 +1,534 @@
+        else:
+            self.status = 'NOT DEPLOIED'
+            logging.error("[Multinet] Failure during deploying")
+            raise Exception('[Multinet] Failure during deploying')
+
+    def get_switches(self):
+        """ Wrapper to the Multinet SB-Emulator get_switches handler
+        """
+        logging.info('[Multinet] get_switches')
+        self.status = 'GETTING SWITCHES'
+
+        if not util.netutil.isfile(self.ip, self.ssh_port, self.ssh_user,
+                                   self.ssh_pass, [self.get_switches_hnd]):
+            logging.error('[Multinet] Get_switches handler does not exist')
+            raise Exception('[Multinet] Get_switches handler does not exist')
+
+        exit_status, output = util.netutil.ssh_run_command(self._ssh_conn,
+            ' '.join([self.__pythonpath_cmd_prefix, 'python',
+                      self.get_switches_hnd, '--json-config',
+                      self.__multinet_config_file_remote_path]),
+            '[Multinet.get_switches_hnd]')
+        if exit_status == 0:
+            self.status = 'GOT SWITCHES'
+            logging.info("[Multinet] Successful got switches")
+            return self.__parse_multinet_output('get_switches_topology_'
+                                                'handler', output)
+        else:
+            self.status = 'NOT GOT SWITCHES'
+            logging.error("[Multinet] Failure during getting switches")
+            raise Exception('[Multinet] Failure during getting switches')
+
+    def get_flows(self):
+        """ Wrapper to the Multinet SB-Emulator get_flows handler
+        """
+        logging.info('[Multinet] get_flows')
+        self.status = 'GETTING FLOWS'
+
+        if not util.netutil.isfile(self.ip, self.ssh_port, self.ssh_user,
+                                   self.ssh_pass, [self.get_flows_hnd]):
+            logging.error('[Multinet] Get_flows handler does not exist')
+            raise Exception('[Multinet] Get_flows handler does not exist')
+
+        exit_status, output = util.netutil.ssh_run_command(self._ssh_conn,
+            ' '.join([self.__pythonpath_cmd_prefix, 'python',
+                      self.get_flows_hnd, '--json-config',
+                      self.__multinet_config_file_remote_path]),
+            '[Multinet. get_flows_hnd]')
+        if exit_status == 0:
+            self.status = 'GOT FLOWS'
+            logging.info("[Multinet] Successful got flows")
+            return self.__parse_multinet_output('get_flows_topology_handler',
+                                                output)
+        else:
+            self.status = 'NOT GOT FLOWS'
+            logging.error("[Multinet] Failure during getting flows")
+            raise Exception('[Multinet] Failure during getting flows')
+
+    def init_topos(self):
+        """ Wrapper to the Multinet SB-Emulator init_topos handler
+        """
+        logging.info('[Multinet] init_topos')
+        self.status = 'INIT_MININET_TOPOS'
```

Comment:
We have a consistency issue here... in other places we use spaces for multi-word statuses, here we use underscores. Let's follow the same approach everywhere (I would opt for underscores). 

Check this for controller class statuses, too.